doc/api/buffer.md

```diff
@@ -103,8 +103,14 @@ string into a `Buffer` as decoding.
 
 * `'base64'`: [Base64][] encoding. When creating a `Buffer` from a string,
   this encoding will also correctly accept "URL and Filename Safe Alphabet" as
-  specified in [RFC 4648, Section 5][]. Whitespace characters such as spaces,
-  tabs, and new lines contained within the base64-encoded string are ignored.
+  specified in [RFC 4648, Section 5][]. Whitespace characters such as
+  spaces, tabs, and new lines contained within the base64-encoded string are
+  ignored.
+
+* `'base64url'`: [base64url][] encoding as specified in
+  [RFC 4648, Section 5][]. When creating a `Buffer` from a string, this
+  encoding will also correctly accept regular base64-encoded strings. When
```

Comment:
Optional nit:

```suggestion
  encoding will also accept regular base64-encoded strings. When
```

Reply:
I've chosen to ignore this nit, the wording is the same in base64.